tests/phpunit/tests/test-admin-static-pages.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+class Admin_Static_Pages_Test extends PLL_UnitTestCase {
+	protected static $home_en;
+
+	/**
+	 * @param PLL_UnitTest_Factory $factory
+	 * @return void
+	 */
+	public static function pllSetUpBeforeClass( PLL_UnitTest_Factory $factory ) {
+		parent::pllSetUpBeforeClass( $factory );
+
+		$factory->language->create_many( 2 );
+		self::$model->options['default_lang'] = 'en'; // Otherwise static model isn't aware of the created languages...
```

Comment:
Since this is a whole new test class, I suggest you don't use any static model at all.
Either use `$factory->pll_model` or create your own instance for the tests.

Reply:
Gomennasai Sensei 🙇 
Done in [5b32e84](https://github.com/polylang/polylang/pull/1616/commits/5b32e84b0bb397c839553c62d1540e78f5b6b232).